en/docs/reference/connectors/cdc-inbound-endpoint/cdc-inbound-endpoint-config.md

```diff
@@ -0,0 +1,175 @@
+# CDC Inbound Endpoint Reference
+
+The following configurations allow you to configure the CDC Inbound Endpoint for your scenario. 
+
+<style type="text/css">
+.tg  {border-collapse:collapse;border-spacing:0;}
+.tg td{font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;}
+.tg th{font-family:Arial, sans-serif;font-size:20px;font-weight:normal;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;}
+.tg .tg-0pky{border-color:inherit;text-align:left;vertical-align:top}
+</style>
+<table class="tg">
+  <tr>
+    <th class="tg-0pky">Parameter</th>
+    <th class="tg-0pky">Default Value</th>
+    <th class="tg-0pky">Description</th>
+  </tr>
+  <tr>
+    <td class="tg-0pky">interval</td>
+    <td class="tg-0pky">-</td>
+    <td class="tg-0pky">The polling interval for the inbound endpoint to execute each cycle. This value is set in milliseconds.</td>
+  </tr>
+  <tr>
+    <td class="tg-0pky">coordination</td>
+    <td class="tg-0pky">true</td>
+    <td class="tg-0pky">This optional property is only applicable in a cluster environment. In a clustered environment, an inbound endpoint will only be executed in worker nodes. If set to <code>true</code> in a cluster setup, this will run the inbound only in a single worker node. Once the running worker is down, the inbound starts on another available worker in the cluster. By default, coordination is enabled.</td>
```

Comment:
Shouldn't this be "In a clustered environment, an inbound endpoint will be executed in all worker nodes."